docs/DeveloperGuide.md

```diff
@@ -564,7 +582,16 @@ testers are expected to do more *exploratory* testing.
       Expected: Similar to previous.
    
    5. Another similar test to try: Deleting a client while archived clients are being shown
-     
+
+### Clearing the list
+
+1. Clearing the client list.
+
+    1. Prerequisites: None.
```

Comment:
Perhaps we can say something like at least one client in active list, and another client in archive (for easier comparison)

Reply:
The thing is we can clear the client list and archived client list without having a client inside. (We did not throw an error if the client list is empty), So im not sure if it really is a prereq or not. 
I know its not a necessity though since AB3 didnt really code it to throw an error.

Reply:
Approving this first but clearing is pending